### PR TITLE
feat: new ui for handling the application through ahjo (hl-1166)

### PR DESCRIPTION
**Pull request by sirtawast**
## Description :sparkles:

This turned out to be a long one. The next steps should be easier as the base is now laid here.

hl-1167 will contain the rest of the functionality, like the fancy HTML editor, page 3 overview and "thank you for submitting"

#### Frontend

- Address some old issues we found in Design, commits marked (prep)
- Use states to get calculator errors to parent node
- Write e2e tests to ensure the old batch process works
- Add button to set new ahjo ui on / off
- Add stepper for application review 
- Break down application review to 3 pages
- Rewrite parts of ApplicationContext to support new 3 step structure
- Load and update decision proposal drafts
- Load decision proposal templates

#### Backend

- Add decision proposal drafting
- Rewrite parts of decision proposal template by @rikuke to better support the frontend
- Minor changes to get more data visible in application endpoint

### CODE REVIEW

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe handler result is __failed__ for https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-youth-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-youth-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe applicant result is __failed__ for https://helsinkilisa-ui-pr2886.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-youth-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe applicant result is __failed__ for https://helsinkilisa-ui-pr2886.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe handler result is __failed__ for https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-youth-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe handler result is __failed__ for https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe handler result is __failed__ for https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-youth-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe handler result is __failed__ for https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-youth-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

File: frontend/benefit/handler/src/components/applicationReview/applicationProcessingView/AplicationProcessingView.tsx
Comment (EmiliaMakelaVincit) [inline frontend/benefit/handler/src/components/applicationReview/applicationProcessingView/AplicationProcessingView.tsx:1]: Random observation: there's a typo in the name of this file.

**Review by EmiliaMakelaVincit — COMMENTED** [review]
Aside from the hardcoded months count in one notification, the FE parts seemed fine.

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe employer result is __failed__ for https://kesaseteli-employer-pr2886.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2886.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2886.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2886.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Pull request closed without merging**